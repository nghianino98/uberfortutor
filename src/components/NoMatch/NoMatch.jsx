import React from "react";
import {Link} from "react-router-dom";
import './NoMatch.scss';
const NoMatch = () => {
  return (
    <div className="notmatch">
      <div className="page-wrapper mid-content-wrapper">
        <div className="number-text"> 404 </div>
        <div className="sub-text"> Xin lỗi, không tìm thấy đường dẫn. </div>
        <Link className="btn btn-homepage" to="/">
          <i className="fas fa-home" />Về trang chủ
    </Link>
      </div>
    </div>
  )
}

export default NoMatch;