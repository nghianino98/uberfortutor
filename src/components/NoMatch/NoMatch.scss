.notmatch{
  .page-wrapper {
    display: flex;
    align-items: center;
    flex-direction: column;
    justify-content: center;
    text-align: center;
    min-height: 90vh;
    background: #fafafa;
    padding-top: 104px;
    padding-bottom: 80px;
  }
  .mid-content-wrapper {
    width: 86%;
    max-width: 960px;
    margin: auto;
}
.number-text {
  font-size: 200px;
  line-height: 150px;
  color: #58a0ef;
}
.sub-text {
  margin-top: 28px;
  font-size: 18px;
  color: #757575;
}
.btn-homepage {
  margin-top: 16px;
  display: block;
  cursor: pointer;
  background-color: #58a0ef;
  color: #fff;
  padding: 10px 32px;
  border-radius: 50px;
  font-weight: 300;
  i{
    margin-right: 8px;
  }
}
}