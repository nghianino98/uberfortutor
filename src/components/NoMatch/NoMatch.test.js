import React from 'react';
import renderer from 'react-test-renderer';
import NoMatch from './NoMatch';
import { BrowserRouter } from 'react-router-dom';

const Wrapper = (props) => (
  <BrowserRouter>
    <NoMatch {...props}/>
  </BrowserRouter>
)

test('test snapShot render component NoMatch', () => {
  const component = renderer.create(
    <Wrapper />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
}); 