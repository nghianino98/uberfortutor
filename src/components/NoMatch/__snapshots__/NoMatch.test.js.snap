// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test snapShot render component NoMatch 1`] = `
<div
  className="notmatch"
>
  <div
    className="page-wrapper mid-content-wrapper"
  >
    <div
      className="number-text"
    >
       404 
    </div>
    <div
      className="sub-text"
    >
       Xin lỗi, không tìm thấy đường dẫn. 
    </div>
    <a
      className="btn btn-homepage"
      href="/"
      onClick={[Function]}
    >
      <i
        className="fas fa-home"
      />
      Về trang chủ
    </a>
  </div>
</div>
`;
